collector/cpu_linux.go

```diff
@@ -54,7 +54,7 @@ func init() {
 func NewCPUCollector() (Collector, error) {
 	return &cpuCollector{
 		cpu: prometheus.NewDesc(
-			prometheus.BuildFQName(namespace, "", cpuCollectorSubsystem),
+			prometheus.BuildFQName(namespace, cpuCollectorSubsystem, "seconds_total"),
```

Comment:
Wouldn't `cpu_usage_seconds_total` be the more common naming here rather than just `cpu_seconds_total`?

Reply:
Maybe, but not everything is `used`, what about `idle` and `steal`?

Reply:
Dunno "used" still sounds ok to me even for those :) But I don't care much.

Reply:
/cc @matthiasr